pkgs/by-name/ve/vencloud/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  nixosTests,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+buildGoModule rec {
+  pname = "vencloud";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "Vencord";
+    repo = "Vencloud";
+    rev = "v${version}";
+    hash = "sha256-u++1qWH04MKipnTGo04FVS+HYcx52AWZpBcjiFMp+mY=";
+  };
+
+  vendorHash = "sha256-4g3mGMhsBaJ4N8SEj56sjAgfH5v8J2RD5c5tMLk5hGU=";
+
+  passthru = {
+    tests = {
+      inherit (nixosTests) vencloud;
+    };
+
+    updateScript = nix-update-script { };
+  };
+
+  meta = with lib; {
+    description = "Cloud settings sync for vencord";
+    homepage = "https://github.com/Vencord/Vencloud";
+    license = licenses.agpl3Only;
+    maintainers = with maintainers; [ eveeifyeve ];
+  };
```

```suggestion
  meta = {
    description = "Vencord's API for cloud settings sync";
    homepage = "https://github.com/Vencord/Vencloud";
    changelog = "https://github.com/Vencord/Vencloud/releases/tag/${src.tag}";
    license = lib.licenses.agpl3Only;
    maintainers = with lib.maintainers; [ eveeifyeve ];
    mainProgram = "backend";
  };
```
Nitpicking on the `with lib`;, but `mainProgram` and `changelog` would be nice here. We can also use the same description from upstream README